lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -566,71 +574,58 @@ def set_depthshade(self, depthshade):
         self._depthshade = depthshade
         self.stale = True
 
-    def set_facecolor(self, c):
-        # docstring inherited
-        super().set_facecolor(c)
-        self._facecolor3d = self.get_facecolor()
-
-    def set_edgecolor(self, c):
-        # docstring inherited
-        super().set_edgecolor(c)
-        self._edgecolor3d = self.get_edgecolor()
-
-    def set_sizes(self, sizes, dpi=72.0):
-        # docstring inherited
-        super().set_sizes(sizes, dpi=dpi)
-        self._sizes3d = self.get_sizes()
-
-    def set_linewidth(self, lw):
-        # docstring inherited
-        super().set_linewidth(lw)
-        self._linewidth3d = self.get_linewidth()
-
```

Comment:
You still need these to update the `_*3d`  versions when the user calls them to modify the collection.

Reply:
I think in 3.3 you could not (easily) set either of these.  

The colors I think "just work", I putting `set_linewidth` back in is straight forward, but due to the call to `set_size` in collections I think we have to do something like

```
def set_size(self, sizes, dpi=72.0)
    if self._internal_mode:
        super().set_size(sizes, dpi)
    else:
        self._sizes3D = sizes

def draw(self, renderer):
    with set_internal_mode(self):
        super().draw()
```

Which thinking about it seemed overly complicated, but typing it out to make the point that it was overly complicated it does not seem so bad after all...

Reply:
These got moved around, but put them back in.